### ISSUE TITLE
Yandex Cloud support

### ISSUE THREAD

**Issue by jokull** [voorhoede/datocms-plugin-translate-fields#45]
I forked and added support for Yandex Cloud (translations 2.0 API). If anyone is interested let me know. There is no CORS support so the request needs to be proxied unfortunately (otherwise I would have a PR).

**Comment by Siilwyn** [comment]
Thoughts on adding support for this with configuration that requires an URL to use as a proxy?

**Comment by jokull** [comment]
Yeah potentially. We're actually moving to Azure later this week since Yandex cloud doesn't support credit cards (lol).

**Comment by jokull** [comment]
Azure has been working quite well for us. They support CORS.

**Comment by velomovies** [comment]
Closed, because we are (at the moment) not going to implement this. Please open a new issue if this is still relevant or necessary
